starter-api/src/main/java/io/micronaut/starter/api/create/github/client/v3/GitHubApiOperations.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.api.create.github.client.v3;
+
+import io.micronaut.http.HttpHeaders;
+import io.micronaut.http.annotation.*;
+import io.reactivex.Flowable;
+
+/**
+ * GitHub Oauth operations.
+ *
+ * @author Pavol Gressa
+ * @since 2.1
+ */
+public interface GitHubApiOperations {
+
+    @Post(value = "/user/repos", single = true)
+    Flowable<GitHubRepository> createRepository(
+            @Header(HttpHeaders.AUTHORIZATION) String oauthToken,
+            @Body GitHubRepository gitHubRepository);
```

Comment:
Since you are justing calling `blockingFirst()` on the results of these methods you can just remove the reactive types since they add no value